target/consensource-ui
# Perichsen/disable bootstrap sourcemap

## Proposed change/fix

Remove `boostrap.min.css` SourceMap import. The import is throwing a warning in DevTools console, and we don't need the SourceMap.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

`npm run watch` and verify that there is no SourceMap warning in the DevTools console.
